### PR TITLE
Post on Dropwizard metrics 

### PR DESCRIPTION
**Pull request by thombergs**

### CODE REVIEW

File: _posts/frameworks/2017-04-21-transparency-with-spring-boot.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+Luckily, there are tools for creating such metrics, so we don't have to
+do it on our own. [Dropwizard metrics](http://metrics.dropwizard.io/) is
+such a tool, which makes it very easy to create metrics within our Java 
+application.
+
+### Injecting the MetricsRegistry
+First off, you will need a `MetricRegistry` object at which to register
+the metrics you want to measure. In a Spring Boot application, you simply
+have to add a dependency to the [Dropwizard metrics library](https://mvnrepository.com/artifact/io.dropwizard.metrics/metrics-core).
```
Comment (matthiasbalke) [inline _posts/frameworks/2017-04-21-transparency-with-spring-boot.md]: You should use a more serious page like: http://search.maven.org/#search%7Cga%7C1%7Cg%3Aio.dropwizard.metrics%20a%3Ametrics-core

or 

http://metrics.dropwizard.io/3.2.2/getting-started.html

File: _posts/frameworks/2017-04-21-transparency-with-spring-boot.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+* the 75th, 95th, 98th, 99th and 999th percentile of the transaction duration
+
+The 99th percentile means that 99% of the measured transactions were faster
+than this value and 1% was slower. Additionally, a `Timer` also creates all 
+metrics of a `Meter`.
+
+## Exposing Metrics via Spring Boot Actuator
+Having measured the metrics, we still need to expose them, so that
+some monitoring tool can pick them up. Using Spring Boot, you can simply
+add a dependency to the [Actuator Plugin](https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-actuator).
```
Comment (matthiasbalke) [inline _posts/frameworks/2017-04-21-transparency-with-spring-boot.md]: also use maven.org

File: _posts/frameworks/2017-04-21-transparency-with-spring-boot.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+}
+```
+
+## Wrap-Up
+When implementing a web application, think of the business metrics you want
+to measure and add a Dropwizard `Meter` or `Timer` to create those metrics.
+It's a few lines of code that provide a huge amount of insight into an
+application running in production. 
+Spring Boot offers first class support for Dropwizard metrics by automatically
+exposing them via the "/metrics" endpoint to be picked up by a monitoring tool.
```
Comment (matthiasbalke) [inline _posts/frameworks/2017-04-21-transparency-with-spring-boot.md]: use code style quotes ` for urls

**Comment by thombergs** [comment]
fixed review issues and merged

**Pull request merged**